### DISCUSSION TITLE
what am I doing wrong dreambooth training

### DISCUSSION THREAD

**Discussion by misiek75** [TheLastBen/fast-stable-diffusion#1073] [Q&A] [answered]
Hi

I am trying to create a style based on vector graphics. But I don't know why the results are very poor. I don't know what I am doing wrong maybe someone can help

I changed cfg, sample method, sampling steps and still the results are poor

![ory_sample](https://user-images.githubusercontent.com/114853131/208293222-2079e207-23ab-4958-9b64-3b37dc85434b.jpg)

settings 1 attempt to train:

file names: omve
Instance Images: 23 
Concept Images: 266

![settings](https://user-images.githubusercontent.com/114853131/208293270-0dfdb4ee-02f7-4dd7-858e-5ec77ed4a1c4.jpg)

vector ilustration of Nissan Skyline r32 by omve
Negative prompt: black & white
Steps: 35, Sampler: Euler a, CFG scale: 7
![vector ilustration of Nissan Skyline r32 by omve_1](https://user-images.githubusercontent.com/114853131/208294039-9cae509f-0719-4958-becf-ada30d041517.png)

settings 2  attempt to train:

file names: omve1
Instance Images: 23 
Concept Images: 266

![settings1](https://user-images.githubusercontent.com/114853131/208294555-b8804116-6e21-4428-bfa2-0272b00cff09.jpg)

vector ilustration of Nissan Skyline r32 by omve1
Negative prompt: black & white
Steps: 35, Sampler: Euler a, CFG scale: 7

![00410-1839366787-vector ilustration of Nissan Skyline r32 by omve1](https://user-images.githubusercontent.com/114853131/208294586-57e5576d-709f-4247-a2b9-23b282bbf65e.png)

**Comment by TheLastBen** [comment, marked as answer]
untick the box "style training" and train the unet for 1300 step, text enc for 350 and set the concept to 0

even though I think it captured the style very well, you just need to play with the prompt to get the right perspective

**Comment by misiek75** [comment]
![00559-994282696-flat vector ilustration center, of bmw z3  color  black, by (omve_1 8) (1)](https://user-images.githubusercontent.com/114853131/208303481-ff5e6d49-c713-4243-a4d1-db3ba6cc32d3.png)

Now it is good :) but upscaling washes all image. What upscaler is good for such graphics ?

**Reply by TheLastBen** [reply]
use img to img if you want a resolution below 1280x1280
